tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,130 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function() {
+	var tests = 1,
+		element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after disabling" );
+	$.each( $.ui.controlgroup.prototype.options.items, function( widget, selector ){
+		element.children( selector ).each(function(){
+			expect( ++tests );
+			strictEqual( $( this )[ widget ]( "widget" ).hasClass( "ui-state-disabled"), true,
```

Comment:
This could be heavily simplified, just search for ".ui-state-disabled", then check that length is 0.